### PR TITLE
machines: Added InfoRecord element, for nice table grouping detail information

### PR DESCRIPTION
**Pull request by bond95**
Previously InfoRecord element was in that PR https://github.com/cockpit-project/cockpit/pull/7872, but I had to move it to separate commit and even pull request, because I would use it for other details (not only detail's modal, but also in tabs). Please review @petervo @andreasn

### CODE REVIEW

File: pkg/machines/components/infoRecord.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React, { PropTypes } from "react";
+
+React;
+
+const InfoRecord = ({id, descr, value}) => {
```
Comment (mareklibra) [inline pkg/machines/components/infoRecord.jsx]: Almost identical component already exists: `<VmOverviewTabRecord/>` in hostvmslist.jsx

File: pkg/machines/components/infoRecord.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React, { PropTypes } from "react";
```
Comment (mareklibra) [inline pkg/machines/components/infoRecord.jsx]: In any case, please add copyright

**Comment by bond95** [comment]
@mareklibra yes, but:
1. It can be used not only in Overview Tab, thus name must be changed.
2. It can be used not only in hostvmslist.jsx, thus it must have separate file
3. Very similar component is already used in vmdiskstab.jsx , and I want to avoid this duplicity in the future.

**Comment by mareklibra** [comment]
@bond95, makes sense. So please refactor within this PR so the component is shared

**Comment by bond95** [comment]
@mareklibra done.

**Comment by petervo** [comment]
I think this needs a rebase to pickup some testing changes.

**Comment by bond95** [comment]
@petervo rebased.

**Comment by petervo** [comment]
@bond95 

Looks like this may be causing a regression.

Seeing alot of these errors in the tests.

```
Page error: Error: React.createElement: unexpect type [ undefined ]
http://127.0.0.2:9391/cockpit/$b66886b9c8f4fe9d2d693238628fe24765578a34/machines/ovirt.js 550 _t
http://127.0.0.2:9391/cockpit/$b66886b9c8f4fe9d2d693238628fe24765578a34/machines/ovirt.js 4364 A
http://127.0.0.2:9391/cockpit/$b66886b9c8f4fe9d2d693238628fe24765578a34/machines/ovirt.js 217 y
http://127.0.0.2:9391/cockpit/$b66886b9c8f4fe9d2d693238628fe24765578a34/machines/ovirt.js 202 g
http://127.0.0.2:9391/cockpit/$b66886b9c8f4fe9d2d693238628fe24765578a34/machines/ovirt.js 67 e
http://127.0.0.2:9391/cockpit/$b66886b9c8f4fe9d2d693238628fe24765578a34/machines/ovirt.js 114 l
http://127.0.0.2:9391/cockpit/$b66886b9c8f4fe9d2d693238628fe24765578a34/machines/ovirt.js 109 u
http://127.0.0.2:9391/cockpit/$b66886b9c8f4fe9d2d693238628fe24765578a34/machines/ovirt.js 67 e
http://127.0.0.2:9391/cockpit/$b66886b9c8f4fe9d2d693238628fe24765578a34/machines/ovirt.js 99 o
http://127.0.0.2:9391/cockpit/$b66886b9c8f4fe9d2d693238628fe24765578a34/machines/ovirt.js 472 at
http://127.0.0.2:9391/cockpit/$b66886b9c8f4fe9d2d693238628fe24765578a34/machines/ovirt.js 472 at
http://127.0.0.2:9391/cockpit/$b66886b9c8f4fe9d2d693238628fe24765578a34/machines/ovirt.js 85 r
http://127.0.0.2:9391/cockpit/$b66886b9c8f4fe9d2d693238628fe24765578a34/machines/ovirt.js 76 n
http://127.0.0.2:9391/cockpit/$b66886b9c8f4fe9d2d693238628fe24765578a34/machines/ovirt.js 290 C
http://127.0.0.2:9391/cockpit/$b66886b9c8f4fe9d2d693238628fe24765578a34/machines/ovirt.js 747 forceUpdate
http://127.0.0.2:9391/cockpit/$b66886b9c8f4fe9d2d693238628fe24765578a34/machines/ovirt.js 331 R
http://127.0.0.2:9391/cockpit/$b66886b9c8f4fe9d2d693238628fe24765578a34/machines/ovirt.js 710 updateComponent
http://127.0.0.2:9391/cockpit/$b66886b9c8f4fe9d2d693238628fe24765578a34/machines/ovirt.js 698 batchUpdate
http://127.0.0.2:9391/cockpit/$b66886b9c8f4fe9d2d693238628fe24765578a34/machines/ovirt.js 362 j
 0 dispatchEvent
phantomjs://code/phantom-lib.js 121 ph_click
```

**Comment by bond95** [comment]
@petervo I did rebase of new master, and for me everything work fine. Try it now.

**Comment by petervo** [comment]
@bond95 same error showing up see here: https://fedorapeople.org/groups/cockpit/logs/pull-7975-20171129-175555-010026f6-verify-fedora-27/log.html#60

**Comment by bond95** [comment]
@petervo fixed.

**Pull request merged**